### ISSUE TITLE
Working with real output

### ISSUE THREAD

**Issue by songeater** [datalogue/keras-attention#37]
Hi - it seems that the original paper and this implementation addresses a target output that is one-hot encoded. To have this work with targets / y-values that are real numbers (I use lstms to experiment with non-quantized audio), would I just have to change the softmax activation in the yt calculation of the step() function below? Eg. change the activation to sigmoid or tanh?

```
        yt = activations.softmax(
            K.dot(ytm, self.W_o)
            + K.dot(st, self.U_o)
            + K.dot(context, self.C_o)
            + self.b_o)
```

Or does the attention concept, as described in the paper, not work with real-valued targets/outputs? I know that lstm models typically function best with quantized data/ one-hot vectors quashed with a softmax function... but real output is what i am playing with.
This is neat work... thanks!

**Comment by zafarali** [comment]
I think it should be fine to predict real valued outputs without the softmax. Make sure to change the loss function to MSE or something more appropriate. Are you having any specific issues?
